src/main/java/zeenea/connector/common/ItemIdentifier.java

```diff
@@ -7,6 +7,8 @@
 /**
  * Represents an identifier for an item, consisting of a list of identification properties.
  *
+ * <p>This ItemIdentifier must be unique for each item in the Zeenea Data Catalog.
```

Comment:
Dans le monde "ZeePath", un path est unique à l'échelle d'une connexion. On souhaite vraiment changer d'échelle ici ?

Reply:
Ma suggestion, considérant qu'on reste isofonctionnels : 
```suggestion
 * <p>This ItemIdentifier must be unique for each item in the data source.
```

Reply:
En effet, l'item identifier doit être unique à l'échelle de la connexion, et ta suggestion est plus pertinente/parlante, c'est modifié !